code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -39,6 +39,7 @@
 		/obj/machinery/apiary,
 		/mob/living/simple_animal/cow,
 		/mob/living/simple_animal/hostile/retaliate/goat,
+		/mob/living/simple_animal/hostile/retaliate/polyp,
```

Comment:
do that thing we discussed

and at mob level make it intent based

Reply:
I'll give it a shot. No guarantees, though. I've never tried writing a proc before.

Reply:
Well, an attempt was made.